bundle/language_en.properties

```
@@ -417,6 +421,7 @@ angal.common.note.txt
 angal.common.ok.btn                                                                                    = OK
 angal.common.ok.btn.key                                                                                = O
 angal.common.opd.txt                                                                                   = OPD
+angal.common.page.label                                                                                = / 0 Page
```

```suggestion
angal.common.page.label                                                                                = Page
```
`/ 0` doesn't need to be in the traduction file